_posts/2020-11-02-dynamically-changeable-logging.md

```diff
@@ -0,0 +1,159 @@
+---
+layout: post
+title:  "Dynamically changeable logging levels"
+date: 2020-11-02
+author: stanislav_knot
+---
+
+Logging is an indisputably important feature for applications.
+Logging monitors what the application is doing, but equally importantly,  it logs what the application is not doing. 
+Looking into application logs can help us prevent an application remaining or even getting into a faulty state.
+
+<!--more-->
+
+
+## Setting logging levels dynamically
+
+There are many frameworks used for logging in Java applications, for example [`Logback`](http://logback.qos.ch/), [`tinylog`](https://tinylog.org/v2/), or [`log4j`](http://logging.apache.org/log4j/2.x/index.html).
+Because `log4j` is used in Kafka, we do use the same framework for Strimzi operator. 
```

Comment:
You do not have to change it just because of me ... but something I noticed. Log4j and log4j2 are almost like two products. So it sounds a bit weird to say _because Kafka is using Log4j1, we decided to use Log4j2_.